cmd/gosec/main.go

```diff
@@ -286,8 +283,9 @@ func main() {
 	}
 
 	// Color flag is allowed for text format
-	if *flagColor && *flagFormat != "text" {
-		logger.Fatalf("cannot set color with %s format. Only text format is accepted", *flagFormat)
+	var color bool
+	if *flagFormat != "text" {
```

Comment:
shouldn't it be `== "text"`, otherwise the color won't be applied

Reply:
Thanks! I fixed it in https://github.com/securego/gosec/pull/463.